game.js

```diff
@@ -0,0 +1,62 @@
+function drawmap(){
+    var canvas = document.getElementById("main");
+    var context = canvas.getContext("2d");
```

Comment:
You can use one var keyword, why did you use two? 
Use var canvas= blablabla,
             context = blablabla;